raiden-dapp/src/store/index.ts

```diff
@@ -70,12 +71,21 @@ const hasNonZeroBalance = (a: Token, b: Token) =>
   (!(a.balance as BigNumber).isZero() || !(b.balance as BigNumber).isZero());
 
 const settingsLocalStorage = new VuexPersistence<RootState>({
-  storage: window.localStorage,
```

Comment:
According to [the docs](https://github.com/championswimmer/vuex-persist#constructor-parameters--) this is the default anyways. So I left it out for my new plugin instance. Since I wanted to keep the code more homogeneous, I removed it here as well.

Reply:
Talked about it in person.